wagtail/admin/views/reports/page_types_usage.py

```diff
@@ -0,0 +1,175 @@
+import django_filters
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.exceptions import PermissionDenied, ValidationError
+from django.db.models import Count, F, OuterRef, Q, Subquery
+from django.utils.translation import gettext_lazy as _
+
+from wagtail.admin.filters import WagtailFilterSet
+from wagtail.admin.views.reports import ReportView
+from wagtail.coreutils import get_content_languages
+from wagtail.models import ContentType, Page, Site, get_page_models
+from wagtail.permission_policies.pages import PagePermissionPolicy
+from wagtail.users.utils import get_deleted_user_display_name
+
+
+def _get_locale_choices():
+    choices = [  # noqa: C416
```

Comment:
Ruff seems to (wrongly) think that this list comprehension is immutable, suggesting to use `list(...)`, which is not appropriate here. We do need the comprehension, hence disabling the linting error.

Reply:
Are you sure we need a list comprehension here? I think ruff is flagging up the fact that `get_content_languages().items()` (or `list(get_content_languages().items())` if it absolutely needs to be a list) gives the same result as the list comprehension.

Reply:
That's absolutely right, @gasman! Thanks, I've fixed it now!